### PR TITLE
Migrate Kovan full-block test to Goerli

### PR DESCRIPTION
**Pull request by feuGeneA**

### CODE REVIEW

File: packages/hardhat-core/src/internal/core/config/default-config.ts [hunk 1]
```diff
@@ -99,6 +99,16 @@ export const defaultHardhatNetworkParams: Omit<
         ]),
       },
     ],
+    [
+      5, // goerli
+      {
```
Comment (fvictorio) [inline packages/hardhat-core/src/internal/core/config/default-config.ts:104]:
```suggestion
      {
        // taken from https://github.com/eth-clients/goerli/
```

Comment (fvictorio) [reply packages/hardhat-core/src/internal/core/config/default-config.ts:104]: (I don't know if you took the block numbers from there, but it's what I used to double-check :sweat_smile:)

**Review by fvictorio — APPROVED** [review]
Minor comment, pre-approving.

File: packages/hardhat-core/src/internal/core/config/default-config.ts (hunk 1 above)
Comment (feuGeneA) [reply packages/hardhat-core/src/internal/core/config/default-config.ts:104]: fyi the source is listed at the beginning of that list of hardfork histories for different networks: https://github.com/NomicFoundation/hardhat/pull/3030/files#diff-12aa9a96d746a7b79bc952eea2f359a0b5c12c87f5215689936e26ececde6d1bR61

Comment (fvictorio) [reply packages/hardhat-core/src/internal/core/config/default-config.ts:104]: Oh right, my bad!

**Pull request merged**